extensions/adobe/experience/decisioning/proposition-response-details.example.1.json

```diff
@@ -0,0 +1,28 @@
+{
+  "https://ns.adobe.com/experience/decisioning/responseDetails": [
+    {
+      "xdm:decisionEventId": "TNT:activity1:experience1:1597208433",
+      "xdm:eventContext": {
+        "xdm:scope": "demo-box",
+        "xdm:activityID": "actitivy1"
```

Comment:
typo: "actitivy1" => "activity1"

Reply:
Fixed. Thanks Jon. I did move the custom extended eventContext to another example. (proposition-response-details.example.2.json)